airflow/README.md

```diff
@@ -0,0 +1,49 @@
+# Airflow Orchestration of Dataproc Templates
+
+Content in this folder demonstrates how to run Dataproc Templates from DAGs running in Airflow
+
+1) Structure your Airflow dags folder following this structure:
+```
+dags/
+    |_submit_dataproc_template_serverless.py
+    |_config/
+            |_submit_dataproc_template_serverless.ini
+    |_dependencies/
+            |_dataproc_templates/
```

Comment:
Structure definition is easily understandable.
But how does someone move `_dataproc_templates/` and it's subdirectories inside dependencies folder ?
In other words how to deploy this structure ?

Reply:
Aah I see you have included some `cp -r` instructions for airflow.
Maybe we can be a bit more descriptive here as well for non-composer deployment

Reply:
Added deployment instructions